src/renderer/src/components/TransactionActivityList/TransactionActivityListEvent.tsx

```diff
@@ -0,0 +1,159 @@
+import { Fragment, ReactNode } from 'react'
+import { useTranslation } from 'react-i18next'
+import { Link } from 'react-router-dom'
+import { StringHelper } from '@renderer/helpers/StringHelper'
+import { TFullTransactionAssetEvent, TFullTransactionEvent, TFullTransactionNftEvent } from '@shared/@types/hooks'
+import { match } from 'ts-pattern'
+
+import { TransactionActivityListTooltip } from './TransactionActivityListTooltip'
+
+type TColumnContentProps = Pick<TColumnProps, 'data'>
+
+type TColumnProps = {
+  data: ReactNode
+  label?: string
+  url?: string
+}
+
+type TAddressColumnDataProps = {
+  address: string
+  addressName?: string
+}
+
+type TProps = {
+  event: TFullTransactionEvent
+}
+
+const ColumnContent = ({ data }: TColumnContentProps) =>
+  typeof data === 'string' || typeof data === 'number' ? (
+    <TransactionActivityListTooltip data={data}>
+      <span className="inline-block max-w-20 truncate">{data}</span>
+    </TransactionActivityListTooltip>
+  ) : (
+    data
+  )
+
+const Column = ({ data, label, url }: TColumnProps) => (
+  <div className="flex flex-col">
+    {label && <p className="font-medium text-gray-300">{label}</p>}
+
+    {url ? (
+      <Link to={url} target="_blank" className="flex text-neon">
+        <ColumnContent data={data} />
+      </Link>
+    ) : (
+      <span className="flex text-white">
+        <ColumnContent data={data} />
+      </span>
+    )}
+  </div>
+)
+
+const AddressColumnData = ({ address, addressName }: TAddressColumnDataProps) => {
+  const tooltipLabel = `${address} ${addressName ? `(${addressName})` : ''}`.trim()
```

Comment:
Separate the condicional in another var
```suggestion
  const addressNameLabel = addressName ? `(${addressName})` : ''
  const tooltipLabel = `${address} ${addressNameLabel}`.trim()
  
  // or
  
  let tooltipLabel = address
  if (addressName) {
    tooltipLabel += ` (${addressName})`
  }
```